### PR TITLE
Added Photo_to_Ascii

### LINKED ISSUE #847: Photo_to_Ascii
I want to work on this issue
Please assign me for this issue under gssoc'21

### PR DESCRIPTION
**Pull request by Avishake007**
# Description

Here I have added photo to ascii feature which will convert your photo to ascii

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

Previously

photo_to_ascii

After

photo_to_ascii(2)

### CODE REVIEW

File: Photo To Ascii/photo_to_ascii.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    # convert image to ascii    
+    new_image_data = pixels_to_ascii(grayify(resized_image))
+    
+    pixel_count = len(new_image_data)  
+    ascii_image = "\n".join([new_image_data[index:(index+new_width)] for index in range(0, pixel_count, new_width)])
+    
+    
+    # save result to "ascii_image.txt"
+    with open("ascii_image.txt", "w") as f:
```
Comment (kaustubhgupta) [inline Photo To Ascii/photo_to_ascii.py:41]:
```suggestion
    with open("./Photo To Ascii/ascii_image.txt", "w") as f:
```

Comment (kaustubhgupta) [inline Photo To Ascii/photo_to_ascii.py:41]: Make file name as "{image_name} (ASCII).txt"

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by Avishake007** [comment]
@kaustubhgupta  It's done . Kindly check it

**Review by kaustubhgupta — APPROVED** [review]

**Review by Kushal997-das — APPROVED** [review]
👍

**Review by santushtisharma10 — APPROVED** [review]
LGTM

**Review by antrikshmisri — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]
LGTM..💯

**Pull request merged**